editor/effects.js

```diff
@@ -113,6 +114,10 @@ export default {
 			) );
 		}
 
+		// Update metaboxes.
```

Comment:
Can we prefill the `editorSettings` object with something like: `{ metaboxes: [ 'normal', 'side' ] }` while loading the editor to know if there are metaboxes and do these calls and show buttons only if the metaboxes exist?

Reply:
Yeah, that is a definite posibility. Detecting the presence of metaboxes is somewhat tricky though, but it should be doable.